injector/cpu_pressure_test.go

```diff
@@ -51,7 +51,7 @@ var _ = Describe("Failure", func() {
 		manager.On("ThreadID").Return(666)
 
 		stresserManager = &stress.StresserManagerMock{}
-		stresserManager.On("TrackInjectorCores", mock.Anything).Return(cpuset.NewCPUSet(0, 1), nil)
+		stresserManager.On("TrackInjectorCores", mock.Anything, mock.Anything).Return(cpuset.NewCPUSet(0, 1), nil)
```

Comment:
Adding unit tests to highlight/validate the new behaviour would be a plus.

Reply:
i'm setting up new tests for `stresserManager` 🙈 i'm unable to include them here (confession: i still should) as it took me some time today with weird errors. I sent this PR in an early state to get implementation feedback 🙇